python_modules/dagster/dagster/_core/definitions/automation_policy_sensor_definition.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any, Mapping, Optional, cast
+
+from dagster._annotations import experimental
+from dagster._core.definitions.asset_selection import CoercibleToAssetSelection
+
+from .asset_selection import AssetSelection
+from .sensor_definition import DefaultSensorStatus, SensorDefinition
+from .utils import check_valid_name, validate_tags
+
+
+@experimental
+class AutomationPolicySensorDefinition(SensorDefinition):
```

Comment:
Despite the name, I was not expecting this to subclass SensorDefinition - I don't love that pattern in the other places that we do it, and i feel like SensorDefinition must make some assumptions that these new "sensors" do not make (for example, that there is a user-specified evaluation function)

Reply:
We would also need to add logic in the sensor daemon to filter out these types of sensors (which means we need to have some way of identifying them from the snapshot)